### ISSUE TITLE
Import doesn't work

### ISSUE THREAD

**Issue by mikestreety** [edleeman17/omnivore-opml-import#2]
Hey,

Thanks so much for opening this up :) I've just found Omnivore and this will save a world of pain.

I've just run through the docs and run the script - the output says it is `:: Success ::` but there is nothing in the Omnivore feeds page.

I've attached the file i'm using to import (I've had to add `.txt` to the end as Github wouldn't allow a XML upload).

[Inoreader Feeds 20240130.xml.txt](https://github.com/edleeman17/omnivore-opml-import/files/14101688/Inoreader.Feeds.20240130.xml.txt)

**Update**:

I've just found if I put `process.stdout.write(d);` on ine 51, I get

```
{"errors":[{"message":"Unexpected server error"}]}
```

Not much help, I know :(

Any pointers you could give would be appreciated. Thanks!

**Comment by edleeman17** [comment]
Hey Mike! I'll take a look for you

**Comment by edleeman17** [comment]
Hey Mike, thanks again for your detailed ticket. Very helpful. There seems to be an issue with passing 'name' in the payload. I've removed this value for now, to unblock you.

I'll have a look and see what has changed on their side to get this added back in.

https://github.com/edleeman17/omnivore-opml-import/pull/3

**Comment by mikestreety** [comment]
Thanks for the swift action @edleeman17 !

I wonder if you need to pass the name at all? When adding a feed via the UI, it gets the name from the feed itself (which most RSS readers do) and then you can edit it. 

I wouldn't expect the name to be passed, but that's just me 😃

**Comment by edleeman17** [comment]
I saw it as an available parameter on the GraphQL payload, so hooked it up anyway as the OPML files contain a name. Just so that if anyone had edited the name in a previous feed reader it would carry over.

But you’re right, it’s not needed for Omnivore!

**Comment by mikestreety** [comment]
Just tried it and works perfectly. Thanks again!
